TestRunner.psm1

```diff
@@ -40,24 +40,42 @@ function Get-ServerMetrics {
     $result
 }
 
-function Duplicate-OdsLogs {
-    $destination = (Join-Path $logFilePath "OdsLogs")
-    Get-ChildItem -Path $destination -Recurse | Remove-Item -Recurse
-    if (!(Test-Path $destination)) { New-Item -ItemType Directory -Force -Path $destination | Out-Null}
-    $logFiles = Get-ChildItem -Path $logFilePath -Recurse -Include *.log -File
-    foreach ($file in $logFiles) {
-        $filePath = $file.ToString().Replace($logFilePath, $destination)
-        $fileDirectory = Split-Path -parent $filePath
-        if (!(Test-Path $fileDirectory)) { New-Item -ItemType Directory -Force -Path $fileDirectory | Out-Null}
-        $file | Copy-Item -Destination $filePath
+function Test-IsLocalhost {
+    param(
+        [Parameter(Mandatory=$True)]
+        [string]
+        $server
+    )
+
+    # When running on localhost, no need to execute a remote command. Substring
+    # check supports use of named instances.
+    return ($server.IndexOf("localhost") -gt -1) `
```

Comment:
A test to compare with $Env:Computername could be added to make this complete, but ok if left alone.